package.json

```diff
@@ -19,9 +22,12 @@
   },
   "homepage": "https://github.com/LambdaSchool/Sprint-Challenge-Node-Express#readme",
   "dependencies": {
+    "express": "^4.17.1",
     "helmet": "^3.18.0",
     "knex": "^0.17.1",
-    "sqlite3": "^4.0.8"
+    "node": "^12.7.0",
+    "nodemon": "^1.19.1",
```

Comment:
node should be globally on your machine so it shouldn't be needed as a dependency. You can check for the version and put the min version that is required to run the application. Don't remember the syntax off the top of my head, but it's only a version declaration not a dependency addition.